sharding/collation.go

```diff
@@ -15,22 +15,16 @@ type Collation struct {
 
 type CollationHeader struct {
 	shardID           *big.Int        //the shard ID of the shard
-	parentHash        *common.Hash    //the hash of the parent collation
 	chunkRoot         *common.Hash    //the root of the chunk tree which identifies collation body
 	period            *big.Int        //the period number in which collation to be included
 	proposerAddress   *common.Address //address of the collation proposer
-	proposerBid       *big.Int        //the reward from proposer to collator for a winning proposal
-	proposerSignature []byte          //the proposer's signature as part of a proposal
```

Comment:
This is still needed

Reply:
What are we going to use proposer bid and signature for?

Reply:
"The shard clients just calculate the collation hash by SHA3(shard_id ++ chuck_root ++ period ++ proposer_address ++ proposer_signature)" from https://hackmd.io/s2Pirp8qQ3q35gSl7EDSNQ?view

We don't need the bid, but we need the signature